src/bindgen/builder.rs

```diff
@@ -12,7 +12,7 @@ use crate::bindgen::library::Library;
 use crate::bindgen::parser::{self, Parse};
 
 /// A builder for generating a bindings header.
-#[derive(Debug, Clone)]
+#[derive(Debug, Default, Clone)]
```

Comment:
I think adding this is a mistake. `Default` doesn't do the same as ::new(), because of `std_types`. Can we disable that clippy warning altogether?